services/ui-src/src/forms/mcpar/reportStructure.ts

```diff
@@ -146,6 +146,11 @@ export default [
             path: "/mcpar/plan-level-indicators/appeals-state-fair-hearings-and-grievances/grievances-overview",
             formId: "dgo",
           },
+          {
+            name: "Grievances Services",
```

Comment:
I think this should be:
```suggestion
            name: "Grievances by Service",
```